packages/flutter/lib/src/material/progress_indicator.dart

```diff
@@ -66,6 +97,20 @@ abstract class ProgressIndicator extends StatefulWidget {
     super.debugFillProperties(properties);
     properties.add(PercentProperty('value', value, showName: false, ifNull: '<indeterminate>'));
   }
+
+  Widget _buildSemanticsWrapper({
+    @required BuildContext context,
+    @required Widget child,
+  }) {
+    return Semantics(
+      container: true,
+      liveRegion: true,
+      enabled: semanticsEnabled,
```

Comment:
This does not seem right? Setting enabled to true will cause TalkBack to announce that something is enabled. Setting it to false will cause it to announce that something is disabled I don't think we want this.